source/semver.d

```diff
@@ -372,7 +372,7 @@ struct SemVer
     /**
      * Compare two $(B different) versions and return the parte they differ on.
      */
-    VersionPart differAt(ref const SemVer other) const scope @safe pure nothrow
+    VersionPart differAt(ref const SemVer other) const scope @safe pure
```

Comment:
With the error message, it's obvious what is wrong: `assert` *is* `nothrow`, because `throw`ing an `Error` is `nothrow`, but `this != other` calls `opEquals` which I am pretty sure is not `nothrow`.